dataeng/jobs/analytics/SnowflakeCollectMetrics.groovy

```diff
@@ -14,13 +14,8 @@ class SnowflakeCollectMetrics {
             NAME: 'snowflake-collect-credit-metrics',
             CRON: '0 * * * *'
         ]
-        Map SnowflakeQueueDepthConfig = [
-            NAME: 'snowflake-collect-queue-depth',
-            CRON: '*/10 * * * *'
-        ]
         List jobConfigs = [
-            SnowflakeWarehouseCreditConfig,
-            SnowflakeQueueDepthConfig
+            SnowflakeWarehouseCreditConfig
```

Comment:
nit: don't we normally leave commas at the end?

Reply:
As we'll probably be moving this remaining job over to the data health telemetry project eventually, I'm leaving this as is.